app/controllers/api/v1/main_controller.rb

```diff
@@ -0,0 +1,16 @@
+class Api::V1::MainController < ApplicationController
```

Comment:
I think we probably could've kept this in the users controller, but I'm fine with it as is.